examples/07_advanced/plot_surface_image_and_maskers.py

```diff
@@ -51,10 +60,13 @@
             cmap="bwr",
             symmetric_cmap=True,
             bg_on_data=True,
+            vmin=vmin,
+            vmax=vmax,
+            bg_map=fsaverage_sulcal,
```

Comment:
force same vmin and vmax across all subplots to have the same range and avoid changes in baseline due to differences between hemispheres

Reply:
also force to have bg_map for esthethics